ts/src/exmo.ts

```diff
@@ -1365,16 +1374,59 @@ export default class exmo extends Exchange {
             } else {
                 request['client_id'] = clientOrderId;
             }
-            params = this.omit (params, [ 'client_id', 'clientOrderId' ]);
         }
-        if ((type === 'stop') || (type === 'stop_limit') || (type === 'trailing_stop')) {
-            const stopPrice = this.safeNumber2 (params, 'stop_price', 'stopPrice');
-            if (stopPrice === undefined) {
+        const leverage = this.safeNumber (params, 'leverage');
+        if (!isSpot && (leverage === undefined)) {
+            throw new ArgumentsRequired (this.id + ' createOrder requires an extra param params["leverage"] for margin orders');
+        }
+        params = this.omit (params, [ 'stopPrice', 'stop_price', 'triggerPrice', 'timeInForce', 'client_id', 'clientOrderId' ]);
+        if ((type === 'stop_buy') || (type === 'stop_sell') || (type === 'stop_limit_buy') || (type === 'stop_limit_sell') || (type === 'trailing_stop_buy') || (type === 'trailing_stop_sell')) {
```

Comment:
this logic is a bit confusing, let's say the user provides `type = 'stop_limit_buy`, we just set the stop_price parameter? not even the type is put there?

Reply:
I updated it

```
%  exmo createOrder ADA/USDT stop_sell sell 10 undefined '{"stopPrice": 0.24, "marginMode": "isolated", "leverage": 3}'
(node:3074) ExperimentalWarning: Custom ESM Loaders is an experimental feature and might change at any time
(Use `node --trace-warnings ...` to show where the warning was created)
2023-09-07T16:24:05.261Z
Node.js: v18.15.0
CCXT v4.0.83
exmo.createOrder (ADA/USDT, stop_sell, sell, 10, , [object Object])
2023-09-07T16:24:08.021Z iteration 0 passed in 510 ms

{
  id: '692862104547551001',
  clientOrderId: undefined,
  datetime: undefined,
  timestamp: undefined,
  lastTradeTimestamp: undefined,
  status: undefined,
  symbol: 'ADA/USDT',
  type: undefined,
  timeInForce: undefined,
  postOnly: undefined,
  side: undefined,
  price: undefined,
  stopPrice: undefined,
  triggerPrice: undefined,
  cost: undefined,
  amount: undefined,
  filled: undefined,
  remaining: undefined,
  average: undefined,
  trades: [],
  fee: undefined,
  info: { order_id: '692862104547551001' },
  fees: [],
  lastUpdateTimestamp: undefined,
  reduceOnly: undefined,
  takeProfitPrice: undefined,
  stopLossPrice: undefined
}
2023-09-07T16:24:08.021Z iteration 1 passed in 510 ms
```